lib/main.dart

```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
+                      fontWeight: FontWeight.bold,
+                      fontSize: 20.0),
+                ),
+              ),
+            ],
+          ),
+          SingleChildScrollView(
+            // this permit can be scroll
             scrollDirection: Axis.horizontal, // in the horizontal axis
             padding: const EdgeInsets.fromLTRB(10.0, 0.0, 10.0, 0.0),
             child: Row(
               children: [
-                const Text(
-                    'Genres',
+                const Text('Genres',
                     style: TextStyle(
                       color: Colors.white,
                       fontSize: 16,
-                    )
+                    )),
+                const SizedBox(
+                  width: 10,
                 ),
-                const SizedBox(width: 10,),
                 Row(
-                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                  children: movie.genres
-                    .map((genre) => Padding(
-                      padding: const EdgeInsets.all(4.0),
-                      child: FilterChip( // i used this to show the genres, normally is used to checkbox
-                        shape: RoundedRectangleBorder(
-                          borderRadius: BorderRadius.circular(40),
-                          side: const BorderSide(width: 2, color: Colors.indigo),
-                        ),
-                        backgroundColor: Colors.black87,
-                        label: Text(
-                          genre,
-                          style: const TextStyle(
-                              color: Colors.white54,
-                              fontSize: 15,
-                          ),
-                        ),
-                        onSelected: (b) {}),
-                    )).toList()),
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: movie.genres
+                        .map((genre) => Padding(
+                              padding: const EdgeInsets.all(4.0),
+                              child: FilterChip(
+                                  // i used this to show the genres, normally is used to checkbox
+                                  shape: RoundedRectangleBorder(
+                                    borderRadius: BorderRadius.circular(40),
+                                    side: const BorderSide(
+                                        width: 2, color: Colors.indigo),
+                                  ),
+                                  backgroundColor: Colors.black87,
+                                  label: Text(
+                                    genre,
+                                    style: const TextStyle(
+                                      color: Colors.white54,
+                                      fontSize: 15,
+                                    ),
+                                  ),
+                                  onSelected: (b) {}),
```

Comment:
if you replace filterchip with a container you wont need this

Reply:
done, i removed it.